actors/common/src/main/java/org/sunbird/metrics/actors/OrganisationMetricsActor.java

```diff
@@ -427,7 +432,9 @@ private String orgCreationResponseGenerator(String periodStr, Map<String, Object
 
   @SuppressWarnings("unchecked")
   private void orgCreationMetrics(Request actorMessage) {
-    ProjectLogger.log("In orgCreationMetrics api");
+    ProjectLogger.log(
+        "OrganisationMetricsActor:orgCreationMetrics  orgCreationMetrics method called.",
```

Comment:
From:
"OrganisationMetricsActor:orgCreationMetrics  orgCreationMetrics method called."

To:
"OrganisationMetricsActor: orgCreationMetrics called."